.circleci/config.yml

```diff
@@ -241,26 +316,36 @@ jobs:
                     rmdir "$extracted_folder"
                   fi
 
-      - run:
-          name: Check for index.html
-          working_directory: ~/project/repository
-          command: |
-            OUTPUT_DIR=.
-            if [ -n "<< pipeline.parameters.OUTPUT_DIR >>" ]; then
-              OUTPUT_DIR="<< pipeline.parameters.OUTPUT_DIR >>"
-            fi
-            if [ -f "$OUTPUT_DIR/index.html" ]; then
-              echo "index.html found in $OUTPUT_DIR"
-            else
-              echo "Error: index.html not found in $OUTPUT_DIR"
-              exit 1
-            fi
+      - when:
+          condition:
+            equal: [false, "<< pipeline.parameters.IS_NEXT_JS >>"]
+          steps:
+            - run:
+                name: Check for index.html
+                working_directory: ~/project/repository
+                command: |
+                  OUTPUT_DIR=.
+                  if [ -n "<< pipeline.parameters.OUTPUT_DIR >>" ]; then
+                    OUTPUT_DIR="<< pipeline.parameters.OUTPUT_DIR >>"
+                  fi
+                  if [ -f "$OUTPUT_DIR/index.html" ]; then
+                    echo "index.html found in $OUTPUT_DIR"
+                  else
+                    echo "Error: index.html not found in $OUTPUT_DIR"
+                    exit 1
+                  fi
 
       - run:
           name: Create Earthfast Bundle
           working_directory: ~/project
           command: |
-            OUTPUT_DIR="<< pipeline.parameters.OUTPUT_DIR >>"
+            # Check if IS_NEXT_JS is true and set OUTPUT_DIR accordingly
+            if [ "<< pipeline.parameters.IS_NEXT_JS >>" = true ]; then
```

Comment:
Is this output dir different?

Reply:
cause I generate our own output folder, not the one generated with npm run build, I have to move some files and folders from there to make it work